lib/electric/migrations/sync.ex

```diff
@@ -0,0 +1,179 @@
+defmodule Electric.Migrations.Sync do
+  @moduledoc """
+
+  """
+
+  alias Electric.Client
+
+  def sync_migrations(app_id, environment, local_bundle) do
+    with {:ok, server_manifest} <- get_migrations_from_server(app_id, environment),
+         {:ok, new_migrations} <- compare_local_with_server(local_bundle, server_manifest),
+         {:ok, msg} <- upload_new_migrations(app_id, environment, new_migrations),
+         {:ok, "ok"} <- apply_all_migrations(app_id, environment) do
+      {:ok, msg}
+    end
+  end
+
+  def get_migrations_from_server(app_id, environment, with_satellite \\ false) do
+    url =
+      if with_satellite do
+        "apps/#{app_id}/environment/#{environment}/migrations?body=satellite"
+      else
+        "apps/#{app_id}/environment/#{environment}/migrations"
+      end
+
+    case Client.get(url) do
+      {:ok, %Req.Response{status: 200, body: data}} ->
+        {:ok, Jason.decode!(data)}
+
+      {:ok, _stuff} ->
```

Comment:
`IO.inspect/1` probably shouldn't stay here in published code, right?

Reply:
thnaks